src/pybind/mgr/ansible/ansible_runner_svc.py

```diff
@@ -12,6 +13,22 @@
 PLAYBOOK_EVENTS = "api/v1/jobs/%s/events"
 EVENT_DATA_URL = "api/v1/jobs/%s/events/%s"
 
+class AnsibleRunnerServiceError(Exception):
+    pass
+
+def handle_requests_exceptions(func):
+    """Decorator to manage errors raised by requests library
+    """
+    @wraps(func)
+    def inner(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except (requests.exceptions.RequestException,
+               requests.exceptions.TooManyRedirects,
```

Comment:
`TooManyRedirects` and `Timeout` both inherent from `RequestException`

Reply:
Fixed. thanks!